config/config.go

```diff
@@ -274,10 +271,11 @@ type Config struct {
 	// OOMUseTmpStorage unused since bootstrap v93
 	OOMUseTmpStorage bool `toml:"oom-use-tmp-storage" json:"oom-use-tmp-storage"`
 
-	// CheckMb4ValueInUTF8, EnableCollectExecutionInfo, Plugin are deprecated.
```

Comment:
Can we change the comment to say this contains deprecated items? That will make it clearer why they are grouped last.